src/import/pull-common.c

```diff
@@ -358,9 +365,18 @@ static int verify_one(PullJob *checksum_job, PullJob *job) {
         line = strjoina(job->checksum, " *", fn, "\n");
 
         p = memmem(checksum_job->payload,
-                   checksum_job->payload_size,
-                   line,
-                   strlen(line));
+                checksum_job->payload_size,
+                line,
+                strlen(line));
```

Comment:
why the reindentaiton above? you only changed whitespace? let's keep patches minimal and avoid this